source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -894,6 +932,21 @@ message TransformationFunctionCall {
 
 }
 
+message TransformationInvertComparisonOperators {
+
+  // A transformation that, given the result id of some
+  // comparison operator (e.g. >, >=, <, !=), inverts it
+  // and inserts a negation instruction to preserve semantics.
+  // Negation instruction will have a result id |operator_id|.
+
+  // Result id of the instruction to invert.
+  uint32 operator_id = 1;
+
+  // Fresh id that will be used by the operator after the inversion.
+  uint32 fresh_id = 2;
+
+}
```

Comment:
Again I don't feel that, by reading this in isolation, I understand what the transformation does.

Is the idea that it replaces the comparison with a different operator, storing the result into a temporary with id `fresh_id`, and then computes the negation of that temporary into the original `operator_id`?

Reply:
Yes. For some instruction with result id `operator_id` that represents a binary operator (`<`, `>`, `<=`) this transformation will replace that instruction's result id with `fresh_id`, invert the opcode (`<` -> `>=`) and insert `OpLogicalNot` instruction with result id |operator_id| below.

Reply:
Updated the comment in fa3db6d.